src/sdsc/__init__.py

```diff
@@ -1009,6 +1009,42 @@ def splitpath(context, path, wantedsegment='filename'):
     else:
         return path
 
+def splitvalueunit(context, value, part='value'):
+    """ From a combination of numeric value and unit (like "100px"), returns
+    either the value or the unit.
+
+    :param str context: context node (ignored)
+    :param str value: numeric value and unit
+    :param str part: wanted part of input value: 'value' or 'unit'
+    :return: numeric value or unit of input value
+    """
+
+    del context # not used
+
+    value = str(value)
+    part = str(part)
```

Comment:
Right, I guess I would need to fix the documentation. What I get from LXML here, I believe is not exactly a string, though it mostly behaves like one.

Otoh, my test cases are indeed using plain strings.